themes/stanford_basic/dist/js/behaviors.js

```diff
@@ -1,40 +1,6 @@
 /******/ (function() { // webpackBootstrap
 /******/ 	var __webpack_modules__ = ({
 
-/***/ 8035:
-/***/ (function() {
-
-var header = document.getElementById('block-stanford-basic-local-tasks');
-var sticky = 0;
-if (header) {
-  sticky = header.getBoundingClientRect().top;
-  window.onscroll = function () {
-    stickyHeaderOnScroll();
-  };
-}
-
-/**
- * Stick the local block tasks to the top of the window.
- */
-function stickyHeaderOnScroll() {
```

Comment:
@imonroe Was this added to VPGE as a custom feature. I looked at old versions of stanford_basic and didn't see a sticky nav in the code.

Reply:
No, that's in stanford_profile: https://github.com/SU-SWS/stanford_profile/blob/60aa0df07293b5f92aa837e8ab2c335be46a8f0c/themes/stanford_basic/src/js/theme/menu/StickyHeaderOnScroll.js#L14
Not custom for vpge.